connector/connectionProperties.js

```diff
@@ -5,13 +5,15 @@
     props["database"] = attr[connectionHelper.attributeDatabase];
     props["dialect"] = "mongosql";
 
-    if (attr["v-log-directory"] == null) {
-        props["LogDir"] = "console";
-    } else {
+    props["LogDir"] = "console";
+    if (attr["v-log-directory"] != null && !attr["v-log-directory"] === "") {
         props["LogDir"] = attr["v-log-directory"];
     }
 
-    props["LogLevel"] = attr["v-loglevel"];
+    props["LogLevel"] = "OFF";
+    if(attr["v-loglevel"] != null && !attr["v-loglevel"] === "") {
```

Comment:
And similarly, here, you want to use : `if(attr["v-loglevel"] != null && attr["v-loglevel"] !== "")`

Reply:
Made this change as well.